node/clean-up-filesystem.sh

```diff
@@ -255,6 +255,7 @@ packages_to_keep=(
   langpacks
   libacl
   libattr
+  libbpf
```

Comment:
Is this needed?

Reply:
The cleanup script complaints about libbpf.so.0.5.0 missing after switching to AlmaLinux base so I added it to the keep list.